packages/delisp-core/__tests__/typescript-declarations.ts

```diff
@@ -0,0 +1,47 @@
+import mkdirp from "mkdirp";
+
+import childProcess from "child_process";
+import path from "path";
+import fs from "fs";
+import { promisify } from "util";
+
+import { readModule } from "../src/module";
+import { compileModuleToString } from "../src/compiler";
+import { inferModule } from "../src/infer";
+import { generateTSModuleDeclaration } from "../src/typescript-generation";
+
+const exec = promisify(childProcess.exec);
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
+
+// Compile a TS file.
+async function tsc(file: string) {
+  try {
+    return exec(`npx tsc "${file}"`);
```

Comment:
I think just `tsc` should work too, at least I think the node path is set automatically. Feels a bit arbitrary to rely on having `npx` installed.

Reply:
Is it set by `yarn` right?  so if I run it with `./node_modules/jest` would fail?  I can change it or just include the full path in here.